crates/evm/src/executor/fork/multi.rs

```diff
@@ -476,13 +465,11 @@ fn create_fork_id(url: &str, num: Option<u64>) -> ForkId {
 /// This will establish a new `Provider` to the endpoint and return the Fork Backend
 async fn create_fork(
     mut fork: CreateFork,
-    retries: u32,
-    backoff: u64,
 ) -> eyre::Result<(CreatedFork, Handler)> {
     let provider = Arc::new(
         ProviderBuilder::new(fork.url.as_str())
-            .max_retry(retries)
-            .initial_backoff(backoff)
+            .max_retry(fork.evm_opts.fork_retries.unwrap())
+            .initial_backoff(fork.evm_opts.fork_retry_backoff.unwrap())
```

Comment:
these unwraps are unsafe,

I think we can add additional builder functions that take an option for convenience.

Reply:
okay, i'll work on it.